code/modules/mob/living/say.dm

```diff
@@ -137,12 +164,13 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 	if(pressure < ONE_ATMOSPHERE*0.4) //Thin air, let's italicise the message
 		spans |= SPAN_ITALICS
 
-	send_speech(message, message_range, src, bubble_type, spans)
+	send_speech(message, message_range, src, bubble_type, spans, message_language=language)
 
 	log_say("[name]/[key] : [message]")
 	return 1
 
-/mob/living/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, list/spans)
+/mob/living/Hear(message, atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans)
+	//world << "[src] Hear(message=[message], speaker=[speaker], message_language=[message_language], raw_message=[raw_message], radio_freq=[radio_freq], spans=[json_encode(spans)])"

```

Comment:
I'll just remove this, things would get too noisy if it was enabled in testing mode.